source/material_model/averaging.cc

```diff
@@ -321,17 +321,14 @@ namespace aspect
        */
       if (out.viscosities.size() > 1)
```

Comment:
The best I can think of is that if you have only one value, averaging it with itself will not yield anything that's terribly useful or different -- it would be a waste of time. Of course, the comment above the code is terribly unhelpful in all ways I can think of. 

If that passes all tests, I'd be ok with simply removing the `if` statement (or, if we'd like to skip the averaging, with updating the comment).